shared/actions/config/index.tsx

```diff
@@ -553,8 +584,10 @@ function* configSaga(): Saga.SagaGenerator<any, any> {
   // MUST go above routeToInitialScreen2 so we set the nav correctly
   yield* Saga.chainAction<ConfigGen.SetNavigatorPayload>(ConfigGen.setNavigator, setNavigator)
   // Go to the correct starting screen
-  yield* Saga.chainAction<ConfigGen.DaemonHandshakeDonePayload | ConfigGen.SetNavigatorPayload>(
-    [ConfigGen.daemonHandshakeDone, ConfigGen.setNavigator],
+  yield* Saga.chainAction<
+    ConfigGen.DaemonHandshakeDonePayload | ConfigGen.SetNavigatorPayload | SettingsGen.LoadedSettingsPayload
+  >(
+    [ConfigGen.daemonHandshakeDone, ConfigGen.setNavigator, SettingsGen.loadedSettings],
```

Comment:
don't change this

Reply:
again im not sure what i should do here instead